cmd/sync/runner.go

```diff
@@ -258,6 +252,30 @@ func (r *runner) processRetagJobs(ctx context.Context, jobCh <-chan retagJob) er
 	}
 }
 
+func (r *runner) processGetTagsJob(ctx context.Context, job getTagsJob) ([]string, error) {
+	var srcTags, dstTags []string
+
+	eg := new(errgroup.Group)
```

Comment:
Now we are reading tags from src and dst in parallel, but this is only a small optimisation.